src/MainWindow.cpp

```diff
@@ -166,6 +166,12 @@ void MainWindow::MessageReceived(BMessage *message)
 			}
 		}
 			break;
+		case IE_MAINWINDOW_MAINMENU_FILE_DOCS__:
+		{
+			// Open a new WebPositive Window
+			system("sh -c \"/boot/system/apps/WebPositive https://github.com/HaikuArchives/Slayer/blob/master/DOCS.txt &\"");
```

Comment:
Instead of hard coding Web+, it'd be better to launch the preferred HTML handler. See https://github.com/HaikuArchives/BurnItNow/blob/master/Source/BurnWindow.cpp#L548 as example.